lucien-ios/StartMyCollection/StartMyCollectionViewController.swift

```diff
@@ -9,9 +9,64 @@
 import UIKit
 
 final class StartMyCollectionViewController: UIViewController {
+
+    @IBOutlet private weak var addBookButton: UIButton!
+    @IBOutlet private weak var welcomeImageView: UIImageView!
+    @IBOutlet private weak var toDashboardButton: UIButton!
+    @IBOutlet private weak var getStartedDescriptionLabel: UILabel!
+    @IBOutlet private weak var welcomeLabel: UILabel!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setUpStyling()
+    }
+
+    func setUpStyling() {
+        setUpWelcomeLabel()
+        setUpGetStartedDescriptionLabel()
+        setUpToDashboardButton()
+        setUpAddBookButton()
+    }
+
+    func setUpWelcomeLabel() {
+        welcomeLabel.font = LucienTheme.Fonts.muliExtraLightItalic(size: CGFloat(48.0))
+        welcomeLabel.textColor = LucienTheme.dark
+        welcomeLabel.addTextSpacing(spacing: 1.2)
+    }
+    
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Trailing Whitespace Violation: Lines should not have trailing whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Atrailing_whitespace)